Delivery-Service.API/Controllers/ProductMVCController.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using DeliverySystem.DAL.Contracts;
+using DeliverySystem.DAL.Models;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Delivery_Service.API.Filters;
+using System;
+
+namespace Delivery_Service.API.Controllers
+{
+    [ServiceFilter(typeof(NewExceptionFilter))]
+    public class ProductMVCController : Controller
```

Comment:
where routing settings?

Reply:
i see in startup

do the same approach
I mean check that u use same approach for API and mcv controllers

choose startup config or attribute approach

Reply:
route should be "mvc/product"